build.gradle.kts

```diff
@@ -63,38 +64,18 @@ tasks.compileJava {
     targetCompatibility = "11"
 }
 
-publishing {
-    publications {
-        create<MavenPublication>("mavenJava") {
-            pom {
-                name.set(rootProject.name)
-                description.set(pluginDescription)
-
-                licenses {
-                    license {
-                        name.set("The Apache License, Version 2.0")
-                        url.set("https://www.apache.org/licenses/LICENSE-2.0.txt")
-                    }
-                }
-                developers {
-                    developer {
-                        name.set("JFrog")
-                        email.set("eco-system@jfrog.com")
-                    }
-                }
-                scm {
-                    connection.set("scm:git:git://github.com/jfrog/artifactory-gradle-plugin.git")
-                    developerConnection.set("scm:git:git@github.com/jfrog/artifactory-gradle-plugin.git")
-                    url.set("https://github.com/jfrog/artifactory-gradle-plugin")
-                }
-            }
-            from(components["java"])
-        }
-    }
+// Build configurations
+val sourcesJar by tasks.registering(Jar::class) {
+    from(sourceSets.main.get().allJava)
+    archiveClassifier.set("sources")
 }
 
-// Build configurations
-tasks.register<Jar>("uberJar") {
+val javadocJar by tasks.registering(Jar::class) {
```

Comment:
I got the following warnings on this task:
> > Task :javadoc
/Users/yahavi/code/artifactory-gradle-plugin/src/main/java/org/jfrog/gradle/plugin/artifactory/extractor/details/PublishArtifactInfo.java:56: warning: no @return
    public Map<QName, String> getExtraInfo() {
                              ^
/Users/yahavi/code/artifactory-gradle-plugin/src/main/java/org/jfrog/gradle/plugin/artifactory/utils/ConventionUtils.java:88: warning: no @param for configuration
    public static void updateConfig(ArtifactoryClientConfiguration configuration, Project project) {
                       ^
/Users/yahavi/code/artifactory-gradle-plugin/src/main/java/org/jfrog/gradle/plugin/artifactory/utils/ConventionUtils.java:88: warning: no @param for project
    public static void updateConfig(ArtifactoryClientConfiguration configuration, Project project) {
                       ^
/Users/yahavi/code/artifactory-gradle-plugin/src/main/java/org/jfrog/gradle/plugin/artifactory/utils/ProjectUtils.java:56: warning: no @return
    public static boolean hasOneOfComponents(Project project, String... componentNames) {
                          ^
/Users/yahavi/code/artifactory-gradle-plugin/src/main/java/org/jfrog/gradle/plugin/artifactory/utils/TaskUtils.java:112: warning: no @return
    public static List<ArtifactoryTask> getAllArtifactoryPublishTasks(Project project) {
                                        ^
5 warnings